Cookbook-ios/googlePlaces/Full Screen default/PlacesViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  PlacesViewController.swift
+//  Cookbook-ios
+//
+//  Created by Diego Quiros on 06/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import GooglePlaces
+
+// For more information visit the places developers documentation at
+//https://developers.google.com/places/ios-sdk/autocomplete#add_an_autocomplete_ui_control
+
+class PlacesViewController: GeneralViewController {
+    @IBOutlet weak var placeLabel: UILabel!
+    
+    @IBAction func onSelectPlacePressed(_ sender: Any) {
+        let autocompleteController = GMSAutocompleteViewController()
+        autocompleteController.delegate = self
+
+        // Specify the place data types to return.
+        let fields: GMSPlaceField = GMSPlaceField(rawValue: UInt(GMSPlaceField.name.rawValue) |
+          UInt(GMSPlaceField.placeID.rawValue))!
+        autocompleteController.placeFields = fields
+        autocompleteController.tableCellBackgroundColor = .standardBackground
+
+        // Specify a filter.
+        let filter = GMSAutocompleteFilter()
+        filter.type = .address
+        autocompleteController.autocompleteFilter = filter
+
+        // Display the autocomplete view controller.
+        present(autocompleteController, animated: true, completion: nil)
+    }
```

Comment:
What do you think about setting all this in a different file, Because when we reuse this in a project the view controller will have a lot of other things

Reply:
I think extracting it to a private controller extension is better and more organized